CONTRIBUTING.md

```diff
@@ -0,0 +1,471 @@
+# Contribution Guidelines
+
+If you are interested in contributing to the Substrate Documentation, please carefully read
+these contribution guidelines.
+
+We lovingly stole these guidelines from [Google's Fuchsia project](https://fuchsia.dev/), and modified them for our needs.
+
+* [Documentation Standards](#documentation-types)
+
+* [Documentation Types](#documentation-types)
+
+* [Documentation Comments](#documentation-types)
+
+* [Documentation Style](#documentation-style)
+
+
+## Documentation Standards
+
+A document about what to document and how to document it for people who create things that need
+documentation.
+
+### Why document?
+
+Substrate is a new framework for building blockchains. Effective documentation allows new people to join and grow the
+project by having all necessary documentation be clear and concise.
+
+### Who is the audience?
+
+The documentation described here is intended to address a technical audience, i.e. those who expect
+to implement or exercise APIs or understand the internal dynamics of the blockchain system. These
+standards are not intended for end-user product documentation.
+
+### What should I document?
+
+Document protocols, introduce essential concepts, explain how everything fits together.
+
+- Conventions: e.g. this document about documentation, code style
+
+- System Design: e.g. network stack, consensus, runtime, assumptions
+
+- APIs: e.g. JSON-RPC, runtime functions
+
+- Protocols: e.g. schemas, encodings, configuration files
+
+- Tools: e.g. `subkey`, `build-spec`, `purge-chain`
+
+- Workflows: e.g. environment set up, test methodologies, where to find various
+  parts, how to get work done
+
+### Where should I put documents?
+
+Documentation that is only for developers working on creating or maintaining
+a specific part of the code should be kept in the same directory as the source code.
+
+Substrate documentation is split into three projects:
+
+* Substrate Recipes - `substrate.dev/recipes/`
+
+   General best practice guidelines on how to develop with Substrate.
+   If you create best practice documentation about about using a specific
+   feature of Substrate, you should create the documentation in the same
+   directory as the other documentation for that specific feature.
+
+* Substrate Rust Documentation - `substrate.dev/rustdocs/`
+
+   Auto-generated Rust documentation from the Substrate repository. See [Documentation Comments](#documentation-comments)
+
+* Substrate Developer Documentation - `substrate.dev/docs/`
```

Comment:
What about substrate.dev/tutorials?

Reply:
Its more of an index no?

The actual tutorials, if in the substrate.dev site, live at `https://substrate.dev/docs/tutorials/*/`

But we know these tutorials also live in other places (something we might want to change?)

Maybe we could consider creating a `codelab` like platform where we place all of our tutorials outside of docusaurus.

Reply:
We should try to keep this path  `substrate.dev/tutorials` to make it easy for a user to remember and get back to a tutorial.

Reply:
imo it's worth linking to [tutorials](https://substrate.dev/tutorials/) until some `codelab` initiative is further along. 

The links for each tutorial might point to different sites/repos, but I don't think that's a big deal. The point of the tutorials tab on substrate.dev is to provide a central place to keep all of these `codelab`-esque examples.

Reply:
Sorry, so what is the consensus here?

I think we agree that `substrate.dev/tutorials/` is not a place "Where should I put documents" (the title of this section).

It acts as an index for where the documents actually live.

@JoshOrndorff if you agree you can resolve this

Reply:
Agreed that tutorials is an index and should be kept in place. Agreed that it's okay to have some tutorial be external links at least in the medium term possibly indefinitely. Agreed that tutorials that live as part of devhub should go in `/docs`.

I think it just feels funny that we're referring to those paths as domain names rather than file paths in the repo. That's not a big deal for me though.